pkg/straceback/straceback.go

```diff
@@ -1008,4 +1008,5 @@ func (sb *StraceBack) Stop() {
 		}
 	}
 	sb.mainProg.Close()
+	sb.annotationPublisher.Clean()
```

Comment:
Shouldn't you test if `annotationPublisher == nil`  or use `withAnnotationPublisher`?

Reply:
You're right. Thanks for catching it up.